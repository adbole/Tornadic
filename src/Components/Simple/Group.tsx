import styled from "@emotion/styled";


export default styled.div({
    display: "grid",
    gap: "inherit",
});
