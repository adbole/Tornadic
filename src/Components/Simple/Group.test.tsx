import { render } from "@testing-library/react"

import Group from "./Group"


test("Matches snapshot", () => {
    const { container } = render(<Group />)

    expect(container).toMatchSnapshot()
})