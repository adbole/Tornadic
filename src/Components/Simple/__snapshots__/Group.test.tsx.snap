// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Matches snapshot 1`] = `
.emotion-0 {
  display: grid;
  gap: inherit;
}

<div>
  <div
    class="emotion-0 emotion-1"
  />
</div>
`;
